CMakeLists.txt

```diff
@@ -227,6 +227,9 @@ endif()
 
 include(CTest)
 if(BUILD_TESTING)
+  if(DEFINED ENV{CONDA_PREFIX})
```

Comment:
Thank you, but can you be more specific about the issue here?
Ideally, there would be a comment here explaining the issue, how it can be reproduced, and more importantly under which conditions this special code-path could be removed.

This looks like a Conda-specific bug, I am not sure Ninja is the right place to deal with this (since it likely affects tons of other projects).